docs/en/deployment/how_to_use_on_kubernetes.md

```diff
@@ -54,6 +55,8 @@ To install Helm, refer to the [Helm Installation Guide](https://helm.sh/docs/int
        accessKey: "<access-key>"
        secretKey: "<secret-key>"
        bucket: "<bucket>"
+       # If you're using Azure blob , please config its connection string, core.chinacloudapi.cn stands for Azure China 
+       envs: "{\"AZURE_STORAGE_CONNECTION_STRING\": \"DefaultEndpointsProtocol=https;AccountName=xxx;AccountKey=xxx;EndpointSuffix=core.chinacloudapi.cn\"}"
```

Comment:
The AccountName should be provided as accessKey, AccountKey should be provide as secretKey.

The bucket should be https://<bucket>.core.chinacloudapi.cn,  then you don't need this environment variable.

Reply:
My juicefs was formatted and config the helm as document says : name & metaurl are required. But juicefs-csi-controller  throws "unable to create azure client, account name should be...". 

My point is  juiefs  document should mention the connection information, whether it be accessKey, AccountKey or AZURE_STORAGE_CONNECTION_STRING environment variable

Reply:
I think we should fix the document here: https://github.com/juicedata/juicefs/blob/main/docs/en/reference/how_to_setup_object_storage.md#azure-blob-storage
use --access-key as an example, `AZURE_STORAGE_CONNECTION_STRING` is not required.

Reply:
We have updated the docs for Azure blob, please check if that work for you.

Different storage vendors use different environment variable to pass secrets, we don't repeat them here. `bucket`, `access-key` and `secret-key` is the way we provide to work for all vendors.

Reply:
Make sense. AK & sk should work.